enterprise/causal-clustering/src/test/java/org/neo4j/causalclustering/discovery/SharedDiscoveryServiceIT.java

```diff
@@ -89,23 +90,26 @@ public void shouldDiscoverCompleteTargetSetWithoutDeadlocks() throws Exception
         }
     }
 
-    private Callable<Void> createDiscoveryJob( MemberId member, DiscoveryServiceFactory disoveryServiceFactory, Set<MemberId> expectedTargetSet ) throws ExecutionException, InterruptedException
+    private Callable<Void> createDiscoveryJob( MemberId member, DiscoveryServiceFactory disoveryServiceFactory,
+            Set<MemberId> expectedTargetSet ) throws ExecutionException, InterruptedException
     {
         Neo4jJobScheduler jobScheduler = new Neo4jJobScheduler();
         jobScheduler.init();
+        ResolutionResolver resolutionResolver = new NoOpResolutionResolver();
 
-        CoreTopologyService topologyService = disoveryServiceFactory.coreTopologyService( config(), null, member,
-                jobScheduler, logProvider, userLogProvider );
+        CoreTopologyService topologyService = disoveryServiceFactory
+                .coreTopologyService( config(), null, member, jobScheduler, logProvider, userLogProvider,
```

Comment:
@martinfurmanski I have reversed a lot of the format changes, what remains are largely improvements or actual changes (with the odd import re-order). I will avoid formatting files like this in the future, thanks